test/compflags/bradc/printstuff/version.goodstart

```diff
@@ -1 +1 @@
- version 1.25.0
\ No newline at end of file
+ version 1.26.0
```

Comment:
It looks like this now ends in a linefeed where it didn't before, and I _think_ these tests are sensitive to this (where I'm guessing that running `start_test test/compflags/bradc/printstuff/version.chpl` would show a failure).

Reply:
Should be resolved